.github/workflows/docs.yml

```diff
@@ -16,7 +16,7 @@ jobs:
 
     - name: Install dependencies
       run: |
-        python -m pip install pip==20.2
+        python -m pip install pip
```

Comment:
@kashif Can you revert these changes so we don't get a conflict in #1786?

Thanks!

Reply:
```suggestion
        python -m pip install pip==20.2
```